.github/workflows/uploadBinaries.yml

```
@@ -99,23 +99,42 @@ jobs:
             -DLLVM_FORCE_ENABLE_STATS=${{ matrix.build_config.stats }} \
             -DCIRCT_RELEASE_TAG_ENABLED=ON \
             -DCIRCT_RELEASE_TAG=${{ github.ref_name }} \
-            -DCMAKE_EXPORT_COMPILE_COMMANDS=OFF
+            -DCMAKE_EXPORT_COMPILE_COMMANDS=OFF \
+            -DCMAKE_INSTALL_PREFIX=`pwd`/../install
           ninja
           ninja check-circt check-circt-unit
+          ninja install
+          cd ..
+
       - name: Display Files
-        run: file build/bin/*
+        run: |
+          file install/*
+          file install/bin/*
+
+      - name: Name Install Directory
+        id: name_dir
+        run: |
+          SANITIZED=$(echo ${{ github.ref_name }} | sed 's/\//-/g')
```

```suggestion
          SANITIZED=$(echo -n ${{ github.ref_name }} | tr '/' '-')
```
(mainly the no newline touchup, consider letting tr do the thing it's good at but sed is more common+familiar so :shrug:, but since this seems to already work maybe nevermind re:newline?)

---

Great suggestions. I think `tr` is better than `sed` for this purpose too anyway.